go/teams/service_helper.go

```diff
@@ -651,3 +661,62 @@ func ChangeTeamSettings(ctx context.Context, g *libkb.GlobalContext, teamID keyb
 
 	return t.PostTeamSettings(ctx, open)
 }
+
+func removeMemberInvite(ctx context.Context, g *libkb.GlobalContext, team *Team, username string, uv keybase1.UserVersion) error {
+	var lookingFor keybase1.TeamInviteName
+	var typ string
+	if !uv.IsNil() {
+		lookingFor = uv.TeamInviteName()
+		typ = "keybase"
+	} else if libkb.CheckEmail.F(username) {
+		lookingFor = keybase1.TeamInviteName(username)
+		typ = "email"
+	} else {
+		ptyp, name, err := team.parseSocial(username)
+		if err != nil {
+			return err
```

Comment:
What's the UX behind this error case, will it be understandable what the error is?

Reply:
```> kbcl -d=0 team remove-member haaa --user=blah@wjwjw
Are you sure? [y/N] y
▶ ERROR Unknown social network: wjwjw```